modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -14,7 +14,7 @@ Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD i
 .Procedure
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the Red Hat OpenShift GitOps Operator is installed.
-. Navigate to the {rh-app-icon} menu -> *Application Stages* -> *Argo CD*. The login page of the Argo CD UI is displayed in a new window.
+. Navigate to the {rh-app-icon} menu -> *Application Stages* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
```

Comment:
this is no longer included under `Application Stages` subsection. It's in its own subsection now under `OpenShift GitOps`, as seen in the screenshot I included previously. Sorry for any confusion on that!

Reply:
Ok, thanks for the feedback. Updated on my commit.